.nvmrc

```
@@ -0,0 +1 @@
+12.13.1
```

I think any version of Node12 should be fine. 
```suggestion
12
```